### ISSUE TITLE
feature request: save column configuration in most recent maps view

### ISSUE THREAD

**Issue by sebeig** [briansd9/exile-diary#117]
I just discovered the feature to reorder/hide/show certain columns in the most recent maps view. 
It is great. 

It would be even better if my configuration would also be saved, so that I don't have to redo it every time I swith to another view or close exile diary.

**Comment by briansd9** [comment]
It should be saved already... can you check if your settings file `%appdata%\exile-diary\settings.json` contains a `columnOrder` key? Something like this (my settings):

`"columnOrder":["date","area","region","level","time","c","kills","encounters"]`

**Comment by sebeig** [comment]
yes, there is columnOrder in there. 
I tested a little, because I am certain it was not saved before, when I wrote this issue, but it is saved now. 

I think I found my error: 
If I change the order, then don't select Apply, but switch to another screen on the left (stash), the current changes to the columns (which were already showing) are not saved. 

If I hit apply, everything seems to work, I was not able to reproduce a bug here. 
sorry for the issue, I think this can be closed.
